src/frontends/src/query_frontend.hpp

```diff
@@ -262,7 +264,7 @@ class RequestHandler {
     std::shared_ptr<Input> input = decode_input(input_type, pt);
```

Comment:
Reminder to remove `model_name` and `model_version` from the JSON format comment on this method.